domains/steake.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Steake",
+    "repo": "https://github.com/Steake/about/",
+    "owner": {
+        "username": "steake",
+        "email": "ollie.steake@gmail.com",
+        "twitter": "officialELTCOIN"
+    },
+    "record": {
+        "CNAME": "steake.github.io"
```

Comment:
404 page.

Reply:
his actual site seems to be on the [/about](https://steake.github.io/about/) page